includes.container/usr/share/vso/hooks/apt-post

```diff
@@ -1,4 +1,10 @@
 #!/bin/bash
+if [ -z $SUDO_UID ]
+then
+    echo "not running with sudo, export the app manually with vso export"
```

Comment:
```suggestion
    echo "not running with sudo, exported the app manually with vso export"
```

If this is displayed after the action has taken place shouldn't it be exported instead of export here.

Reply:
No, this is in case the user did not use sudo, in which case it's not possible to export so the script just prints this and exits.